dipy/viz/horizon/tab/base.py

```diff
@@ -162,7 +163,7 @@ def __init__(
                 "Images are of different dimensions, "
                 + "synchronization of slices will not work"
             )
-            warnings.warn(msg, stacklevel=2)
+            logging.warning(msg, stacklevel=2)
```

Comment:
stacklevel ?

Reply:
@skoudoro updated the stacklevel as per the discussion.